examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
```

Comment:
I'd consider https://github.com/Phrogz/SLAXML which is pure Lua with no c dependency.

Both are kind of not maintained, so it does not matter much (most active around 2014).